src/Security/Handler/AclSecurityHandlerInterface.php

```diff
@@ -95,18 +116,22 @@ public function deleteAcl(ObjectIdentityInterface $objectIdentity);
     /**
      * Helper method to find the index of a class ACE for a role.
      *
+     * NEXT_MAJOR: change signature to `findClassAceIndexByRole(MutableAclInterface $acl, string $role): array-key|false`
```

Comment:
we should remove all the union types from next major comments, they are only allowed from php 8.0

Reply:
We'll still be able to write `@return array-key|false`, that's what I understand with this comment.

Reply:
> We'll still be able to write @return array-key|false, that's what I understand with this comment.

Yes, that's why I did it like that.